### PR TITLE
Docs  - Minor edits after a test run

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

After meeting this morning with Antonio Nardela,  Holger Weber and @anistark , the following edits and updates came up to make the docs clearer. 

- Minor edits and updates after a test run.

## Type of change

- Enhancement (a non-breaking change that adds functionality)
- Documentation Fix

## How the change has been tested

Docs site was mounted locally.

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by lucas-tortora** [comment]
@anistark could you review this?

File: documentation/docs/guide/chains_and_nodes/running-a-node.md [hunk 1]
```diff
@@ -41,6 +42,13 @@ plugin enabled. Being an experimental plugin, it is currently disabled by defaul
 be enabled via configuration.
 
 :::
+## Download Wasp
+
+You can download the latest Wasp version from the [official repository](https://github.com/iotaledger/wasp).
```
Comment (jorgemmsilva) [inline documentation/docs/guide/chains_and_nodes/running-a-node.md]: this kind of implies that there is a binary to be "downloaded"

```suggestion
You can get the source code of the latest Wasp version from the [official repository](https://github.com/iotaledger/wasp).
```

File: documentation/docs/guide/chains_and_nodes/running-a-node.md [hunk 2]
```diff
@@ -205,3 +230,11 @@ wasp
...
 
 Repeat this process to launch as many nodes as you want for your committee.
+
+### Accessing Your Node From a Remote Machine 
+
+If you want to access your Wasp node outside your local network, you will need to add your IP to the `webpi.adminWhitelist`. You can do so by adding it to your config file, or running the node with the  `webapi.adminWhitelist` flag.
+
+```bash
+wasp --webapi.adminWhitelist=127.0.0.1,YOUR_IP
+```
```
Comment (jorgemmsilva) [inline documentation/docs/guide/chains_and_nodes/running-a-node.md:240]:
```suggestion
If you want to access the Wasp node from outside its local network, you will need to add your public IP to the `webpi.adminWhitelist`. You can do so by adding it to your config file, or running the node with the  `webapi.adminWhitelist` flag.

```bash
wasp --webapi.adminWhitelist=127.0.0.1,YOUR_IP
```
```

File: documentation/docs/guide/chains_and_nodes/running-a-node.md [hunk 3]
```diff
@@ -41,6 +42,13 @@ plugin enabled. Being an experimental plugin, it is currently disabled by defaul
...
 :::
+## Download Wasp
+
+You can download the latest Wasp version from the [official repository](https://github.com/iotaledger/wasp).
+
+```bash
+git clone https://github.com/iotaledger/wasp
+```
 
 ## Compile
```
Comment (jorgemmsilva) [inline documentation/docs/guide/chains_and_nodes/running-a-node.md:52]: should we mention here docker as an alternative?

File: documentation/docs/guide/chains_and_nodes/running-a-node.md [hunk 4]
```diff
@@ -205,3 +230,11 @@ wasp
 You can verify that your node is running by opening the dashboard with a web browser at [`127.0.0.1:7000`](http://127.0.0.1:7000) (default url).
 
 Repeat this process to launch as many nodes as you want for your committee.
+
+### Accessing Your Node From a Remote Machine 
+
+If you want to access your Wasp node outside your local network, you will need to add your IP to the `webpi.adminWhitelist`. You can do so by adding it to your config file, or running the node with the  `webapi.adminWhitelist` flag.
```
Comment (jorgemmsilva) [inline documentation/docs/guide/chains_and_nodes/running-a-node.md]:
```suggestion
If you want to access the Wasp node from outside its local network, you will need to add your public IP to the `webpi.adminWhitelist`. You can do so by adding it to your config file, or running the node with the `webapi.adminWhitelist` flag.
```

File: documentation/docs/guide/chains_and_nodes/running-a-node.md (hunk 3 above)
Comment (lucas-tortora) [reply documentation/docs/guide/chains_and_nodes/running-a-node.md:52]: There's a link to https://wiki.iota.org/wasp/misc/docker on line 16 (first paragraph), but if you think it's worth mentioning again we can certainly add another link here.

Comment (jorgemmsilva) [reply documentation/docs/guide/chains_and_nodes/running-a-node.md:52]: okay then, thanks! - no worth it then, just something that came to mind

File: documentation/docs/guide/chains_and_nodes/running-a-node.md [hunk 5]
```diff
@@ -30,6 +30,7 @@ If you prefer, you can also configure a node [using a docker image](../../misc/d
 
 - [Go 1.16](https://golang.org/doc/install)
 - [RocksDB](https://github.com/facebook/rocksdb/blob/master/INSTALL.md)
+- [Rust](https://www.rust-lang.org/tools/install)
```
Comment (jorgemmsilva) [inline documentation/docs/guide/chains_and_nodes/running-a-node.md]: rust is not needed to run a node, only for creating smart contracts.

**Review by jorgemmsilva — APPROVED** [review]

**Pull request merged**